ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml

```diff
@@ -263,6 +263,22 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-gcp-ipi-xpn-cco-manual-users-static
+- as: ibmcloud-ipi-private-reboot-nodes-f28
+  cron: 44 3 12 * *
+  steps:
+    cluster_profile: ibmcloud-qe
+    env:
+      ENABLE_REBOOT_CHECK: "true"
+    test:
+    - chain: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-ibmcloud-ipi-private-with-ep-cos
```

Comment:
suggest to rename job name to reflect the workflow `with-ep-cos`

Reply:
new a workflow for the reboot, do not use the existed workflow?